store/src/java-test/com/zimbra/cs/mailbox/calendar/AllDayAcceptInstanceTest.java

```diff
@@ -93,13 +109,125 @@ public MailSender getMailSender() {
             }
         });
     }
-    
+
+    @Before
+    public void setUp() throws Exception {
+        System.out.println(testName.getMethodName());
+        MailboxTestUtil.clearData();
+    }
+
+    /*
+     * Test with organizer timezone GMT+
+     * In this case GMT+5:30 Asia/Kolkata
+     */
+    @Test
+    public void testZCS2655PositiveTimeZone() throws Exception {
```

Comment:
I assume large chunks of this test are the same as the next one.  Would be good to share code as much as possible to ensure they stay in sync and make it clear exactly what the differences are.